accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -417,10 +417,15 @@ read.
     is treated as a first read and the initializer expression is evaluated
     again.
 
-A write to a field or variable which is marked `late` and `final` is a runtime
-error unless the field or variable was declared with no initializer expression,
-and there have been no previous writes to the field or variable (including via
-an initializing formal or an initializer list entry).
+Let `v` be a `late` and `final` non-local variable without an initializing
```

Comment:
I'm not sure that I agree with this rewording, at least without being more explicit what we mean by "the setter".  This can be read as "to invoke the setter named `v=`", which I don't think is correct.  Consider:

```
class A {
  late final int v;
}
class B extends A {
  int v;
  B() {
    super.v = 3;
    v = 4; // This is an invocation of `v=`, but it is not an error.
  }
}
```

I'm thinking perhaps think the clearest way to do this is to specify this in terms of the induced setters, getters, and backing store.

Reply:
Just made it explicit which setter causes the run-time error; how do you like that?

Reply:
Sure, I can live with that.  I may follow up.

Reply:
Sounds good, thanks! Then I'll land it if Lasse is happy.